src/services/item/plugins/shortLink/schemas.ts

```diff
@@ -43,57 +46,55 @@ export const getAllByItem = {
   operationId: 'getShortLinksForItem',
   tags: ['shortLink'],
   summary: 'Get all short links for item',
-  description: 'Get all short links information for item.',
+  description:
+    'Get all short links created for an item. The response could be an empty object or a key-value with at least one platform and the alias.',
 
   params: customType.StrictObject({
     itemId: customType.UUID(),
   }),
-  response: { [StatusCodes.OK]: Type.Array(shortLinkPayload), '4xx': errorSchemaRef },
+  response: {
+    [StatusCodes.OK]: Type.Union([
+      Type.Object(
```

Comment:
You could use `customType.StrictObject` to avoid repeating `additionalProperties: false`